packages/test/test-end-to-end-tests/src/test/gc/gcReferenceUpdatesInLocalSummary.spec.ts

```diff
@@ -79,6 +80,7 @@ describeFullCompat("GC reference updates in local summary", (getTestObjectProvid
 
     const runtimeOptions: IContainerRuntimeOptions = {
         summaryOptions: {
+            disableSummaries: true,
```

Comment:
I wonder why/how this test current passes in fullcompat, without the disableSummaries

Reply:
That's because it uses a local container runtime factory. So, even in N-1 / N-2 mode, it creates the current version of container runtime. In other words, it doesn't really test back compat.
I called this out in the PR description and we need update to the test infra to support tests that need custom data stores / container runtime factory.

Reply:
https://dev.azure.com/fluidframework/internal/_workitems/edit/756